hatchways/team-bluejay
# Fe searching for chefs

Test out backend for searching chefs.

Steps to test:
- go to /chefs, you should see all chefs
- apply filter on cuisine. chef has stub cuisines of ""Japanese, Chinese, Mexican"". So, choosing those that do not apply should return with no chefs.
- apply distance filter by typing in Toronto, CA. Chef has stub location around the area of toronto.
- Choosing a filter of 1 should return no chefs. But if slider is put to 0. The filter should not apply, and will return all chefs
